lib/list.js

```diff
@@ -0,0 +1,18 @@
+const { getAPIKey } = require('./userConf');
+const { list } = require('../sdk');
+
+/**
+ * Run performance measurements on deployed function
+ *
+ * @param {string} funcName - name of the function to invoke
```

Comment:
This was my bad, I copied the doc from `perf` as a template and didn't update it.